packages/browser/package.json

```diff
@@ -27,14 +27,14 @@
         "test-watch": "jest --watch src",
         "test:typecheck": "cd src/__tests__ && tsc --noEmit --project tsconfig.json",
         "typecheck": "tsc --noEmit --project tsconfig.json",
-        "playwright": "pnpm exec playwright test --project webkit --project firefox --project chromium",
+        "playwright": "pnpm exec playwright test --project webkit --project firefox --project chromium --grep-invert 'ingestion'",
```

Comment:
i wish json allowed comments
i'm worry that nobody will know what `grep-invert ingestion` does

i wonder if it's easier to have two commands that remove the need to do that

Reply:
Could run a bash file with comments